corona-warn-app/cwa-app-android
# End of life behaviour UI (EXPOSUREAPP-4641)

This PR adds the new card for re-enabling the risk calculation after a user was found positive and has submitted their keys.

Testing steps:
- Register a positive test.
- Submit your keys.
- On the home screen, under the "Vielen Dank" card, the new card that lets you re-enable risk calculation is displayed. 
Note: nothing will happen after pressing the new card. The logic will be implemented in a separate PR.
